staging/src/k8s.io/client-go/Godeps/Godeps.json

```diff
@@ -247,6 +247,10 @@
 			"Rev": "f6abca593680b2315d2075e0f5e2a9751e3f431a"
 		},
 		{
+			"ImportPath": "github.com/stretchr/testify/require",
```

Comment:
It contains `github.com/stretchr/testify/assert` but not `github.com/stretchr/testify/require`. Rebased anyway.